ceph-releases/kraken/fedora/24/daemon/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# DOCKER-VERSION 1.0.0
```

Comment:
Seems as though Ceph is still at Jewel for Fedora. Do we want to remove for now?